src/components/data-table-v2/_data-table-v2-inline-edit-cell.scss

```diff
@@ -0,0 +1,355 @@
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/layout';
+
+@mixin assistive-text {
```

Comment:
Should we merge this to `.#{$prefix}--visually-hidden`?

Reply:
Definitely can, would prefer if the class name is updated, or the styles extended, to include the assistive text content in it just to help better indicate what it is being used for.

Reply:
Right, which is why I just opted for this mixen given the scope of what that change might entail and the lack of time to pursue it 😞

Reply:
Oh I meant something like `@mixin assistive-text { @include visually-hidden; }` - Would it make sense...? Thanks!

Reply:
Gotcha! That makes sense. Is `visually-hidden` a mixin currently, or a class?

Reply:
Currently in [a bulk mix-in](https://github.com/carbon-design-system/carbon-components/blob/5888289/src/globals/scss/_css--helpers.scss). I think we'd separate out `visually-hidden` and include it in `css-helpers` as well as `assistive-text`.

Reply:
Any chance we could introduce this change in a future PR? Don't really want to touch those areas of the codebase with this one.

Reply:
OK - Sounds fine!